src/js/plugins/utils.js

```diff
@@ -4,36 +4,8 @@ export const getPluginName = function(url) {
     return url.replace(/^(.*\/)?([^-]*)-?.*\.(js)$/, '$2');
 };
 
-export function mapPluginToCode(pluginUrl) {
-    let code = 305000;
-    if (!pluginUrl) {
-        return code;
-    }
-
-    switch (getPluginName(pluginUrl)) {
-        case 'jwpsrv':
-            code = 305001;
-            break;
-        case 'googima':
-            code = 305002;
-            break;
-        case 'vast':
-            code = 305003;
-            break;
-        case 'freewheel':
-            code = 305004;
-            break;
-        case 'dai':
-            code = 305005;
-            break;
-        case 'gapro':
-            code = 305006;
-            break;
-        default:
-            break;
-    }
-
-    return code;
+export function mapPluginToCode(/* pluginUrl */) {
```

Comment:
It might make sense to rename this to `getPluginErrorCode` since the mapping is no longer happening here, and we shouldn't need to comment on commercial features here.